### PR TITLE
EASY 1597

### PR DESCRIPTION
**Pull request by gulcinermis**
Fixes EASY-1597

#### When applied it will
* The command "run.sh --version" or "run-service.sh --version"
* generates the following output: 
* easy-collect-curation-work v${project.version} 
Note: In easy-split-multi-deposit, the same commands generate the following output:
easy-split-multi-deposit v${project.version} 

#### Related pull requests on github
repo                       | PR                | note
-------------------------- | ----------------- | ----
easy-                      | [PR#](PRlink)     | ..

### CODE REVIEW

File: src/main/scala/nl.knaw.dans.easy.ccw/CommandLineOptions.scala [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.ccw
+
+import org.rogach.scallop.ScallopConf
+
+class CommandLineOptions(args: Array[String], version: String) extends ScallopConf(args) {
```
Comment (rvanheest) [inline src/main/scala/nl.knaw.dans.easy.ccw/CommandLineOptions.scala]: Most projects have a [`ReadmeSpec`](https://github.com/DANS-KNAW/easy-pid-generator/blob/master/src/test/scala/nl/knaw/dans/easy/pid/ReadmeSpec.scala) to check that the command line is in sync with the README. Can you add that one as well?

File: src/main/scala/nl.knaw.dans.easy.ccw/CommandLineOptions.scala [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  appendDefaultToDescription = true
+  editBuilder(_.setHelpWidth(110))
+  printedName = "easy-collect-curation-work"
+  private val SUBCOMMAND_SEPARATOR = "---\n"
+  val description = ""
```
Comment (rvanheest) [inline src/main/scala/nl.knaw.dans.easy.ccw/CommandLineOptions.scala]: Please add the description here. (see README)

File: src/main/scala/nl.knaw.dans.easy.ccw/CommandLineOptions.scala (hunk 1 above)
Comment (gulcinermis) [reply src/main/scala/nl.knaw.dans.easy.ccw/CommandLineOptions.scala]: Okay, in this case I didn't get any failure with mvnci related to Readme file

Comment (rvanheest) [reply src/main/scala/nl.knaw.dans.easy.ccw/CommandLineOptions.scala]: that's right, because the `ReadmeSpec` isn't there yet...

**Pull request merged**